### PR TITLE
Ports TG tend wounds 

### PR DESCRIPTION
**Pull request by ghost**
i literally just copy pasted the entire file from tg because nothing from it seems to conflict at all. i'm a coder!

anyway this means tend wounds can be performed with clothes on for reduced effectiveness, heals more when the patient is hurt, and can be performed a little easier. 

this is the only good change tg has ever made

https://github.com/tgstation/tgstation/pull/46512

#### Changelog

:cl:  
tweak: adam copy pasted cobby's tend wound code so now tend wounds can be done with clothes on
/:cl:

### CODE REVIEW

**Comment by Ratvar** [comment]
Good because laying on surgeon's table while he has tools out and **you can't actually see what surgery steps he's doing, yes, it's a feature, he can remove your heart instead** is still dangerous, and removing/putting all clothing on makes all surgeries even more of a slog and not worth it then they already are
nor this contends with our OP chemistry system: it's a lame replacement

File: code/modules/surgery/healing.dm [hunk 1]
```diff
@@ -49,107 +52,164 @@
...
 /datum/surgery/healing/burn/upgraded
+	name = "Tend Wounds (Burn, Adv.)"
 	replaced_by = /datum/surgery/healing/burn/upgraded/femto
 	requires_tech = TRUE
 	healing_step_type = /datum/surgery_step/heal/burn/upgraded
+	desc = "A surgical procedure that provides advanced treatment for a patient's burns. Heals more when the patient is severely injured."
 
 /datum/surgery/healing/burn/upgraded/femto
+	name = "Tend Wounds (Burn, Exp.)"
 	replaced_by = /datum/surgery/healing/combo/upgraded/femto
```
Comment (alexkar598) [inline code/modules/surgery/healing.dm:150]:
```suggestion
 	replaced_by = null
```

Comment (ghost) [reply code/modules/surgery/healing.dm:150]: pardon

**Comment by Amelia0010** [comment]
put the TG PR in your PR body, and their name in the changelog :)

File: code/modules/surgery/healing.dm (hunk 1 above)
Comment (Amelia0010) [reply code/modules/surgery/healing.dm:150]: @alexkar598 I don't get why?

Comment (alexkar598) [reply code/modules/surgery/healing.dm:150]:
```/datum/surgery/healing/combo/upgraded/femto //no real reason to type it like this except consistency, don't worry you're not missing anything
	name = "Tend Wounds (Mixture, Exp.)"
	replaced_by = null
```

Its used like this elsewhere,it doesnt seem like a bad idea to not make a thing be replaced by itself. Sounds runtime/issue prone

Comment (Amelia0010) [reply code/modules/surgery/healing.dm:150]: it's not itself though, `/datum/surgery/healing/burn/upgraded/femto` gets replaced by `/datum/surgery/healing/combo/upgraded/femto`

Comment (ghost) [reply code/modules/surgery/healing.dm:150]: this is stupid and you misread

**Pull request merged**